### ISSUE TITLE
Unusable when computer has no audio interface

### ISSUE THREAD

**Issue by broady** [magefree/mage#1941]
My computer has no sound output device (I won't go into why...)

None of the xmage UI works. I see the following exceptions in my terminal:

```
java.lang.NoClassDefFoundError: Could not initialize class mage.client.util.audio.AudioManager
    at org.mage.card.arcane.CardPanel.update(CardPanel.java:836)
    at org.mage.card.arcane.CardPanel.update(CardPanel.java:1164)
    at mage.client.game.BattlefieldPanel.update(BattlefieldPanel.java:188)
    at mage.client.game.PlayAreaPanel.update(PlayAreaPanel.java:564)
    at mage.client.game.GamePanel.updateGame(GamePanel.java:780)
    at mage.client.game.GamePanel.updateGame(GamePanel.java:681)
    at mage.client.remote.CallbackClientImpl$1.run(CallbackClientImpl.java:280)
    at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
    at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:756)
    at java.awt.EventQueue.access$500(EventQueue.java:97)
    at java.awt.EventQueue$3.run(EventQueue.java:709)
    at java.awt.EventQueue$3.run(EventQueue.java:703)
    at java.security.AccessController.doPrivileged(Native Method)
    at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:76)
    at java.awt.EventQueue.dispatchEvent(EventQueue.java:726)
    at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
    at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
    at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
    at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

**Comment by broady** [comment]
This is the command I was using to run xmage:
```
$ java -Xms256m -Xmx512m -XX:MaxPermSize=256m -XX:+UseConcMarkSweepGC -XX:+CMSClassUnloadingEnabled -jar ./lib/mage-client-1.4.11.jar
```

The class is there:

```
$ zipinfo lib/mage-client-1.4.11.jar  | grep AudioM
-rw-r--r--  2.0 unx      926 bl defN 25-Apr-16 23:33 mage/client/util/audio/AudioManager$1.class
-rw-r--r--  2.0 unx     8463 bl defN 25-Apr-16 23:33 mage/client/util/audio/AudioManager.class
```

**Comment by broady** [comment]
Another related exception when I try to watch a currently playing game:
```
java.lang.ExceptionInInitializerError
	at mage.client.game.BattlefieldPanel.update(BattlefieldPanel.java:200)
	at mage.client.game.PlayAreaPanel.update(PlayAreaPanel.java:564)
	at mage.client.game.PlayAreaPanel.<init>(PlayAreaPanel.java:112)
	at mage.client.game.GamePanel.addPlayers(GamePanel.java:624)
	at mage.client.game.GamePanel.init(GamePanel.java:594)
	at mage.client.remote.CallbackClientImpl$1.run(CallbackClientImpl.java:189)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:756)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.awt.EventQueue$3.run(EventQueue.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:76)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:726)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
Caused by: java.lang.IllegalArgumentException: Mixer not supported: null
	at javax.sound.sampled.AudioSystem.getMixer(AudioSystem.java:259)
	at mage.client.util.audio.LinePool.<init>(LinePool.java:52)
	at mage.client.util.audio.LinePool.<init>(LinePool.java:46)
	at mage.client.util.audio.AudioManager.<init>(AudioManager.java:59)
	at mage.client.util.audio.AudioManager.<clinit>(AudioManager.java:58)
	... 20 more
```

**Comment by JayDi85** [comment]
Bloated audio logs was fixed via fbca0f4fef61e82832c8c98f3ac1515f185a7da8. XMage playable without sound.
